docs/release_notes/v0.2.2.md

```diff
@@ -0,0 +1,130 @@
+SMQTK v0.2.2 Release Nodes
+==========================
+This is a minor release. This release primarily adds classifier algorithm
```

Comment:
How about:

Version 0.2.2 is a minor release that primarily added classifier algorithm and classification representation support, a new service web application for nearest-neighbors algorithms, and additional documentation.

Also, this release added few more command line tools, especially of note is iqrTrainClassifier.py that can train a classifier based on the saved state of the IQR demonstration application (also a new feature).